### PR TITLE
Javascript Typing Test (workshop)

### PR DESCRIPTION
**Pull request by GurshanBrar**
Hello there! This is my second pull request, so obviously I have to thank you for the dedication you put into reviewing this for me.

The link to the mock meeting:
https://hackclub.slack.com/archives/C01A9V5LPD0/p1609467151063400

We create a small little typing speed test in this workshop. It tells you the WPM, accuracy, and time.

### CODE REVIEW

File: workshops/javascript_typing_test/README.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+Inside the ```<div>``` there is an ```<h1>``` and a ```<p>```. The ```<h1>``` tag defines an HTML heading. The ```<p>``` tag represents a text. 
+
+You can see how we have a weird class called ```font-weight-light```. Well it turns out, there's another great property of ```class``` in our case. Remember that Bootstrap stylesheet we used? That stylesheet contains custom styles that are referenced under class names. Using those class names in our HTML will allow the stylesheet to select our element and the styles will be applied. 
+
+Therefore, a class we use is [```font-weight-light```](https://getbootstrap.com/docs/4.5/utilities/text/#font-weight-and-italics). This class means that the text will have a *light* look and feel to it. It allows us to create different styles very easily and we don't have to code any CSS for this purpose!
+
+To create the container to hold our text for typing, we use this code:
+```html
+<div id="main" class="container main">
+		<div class="text-container" id="text-container"></div>
```
Comment (giridhar7632) [inline workshops/javascript_typing_test/README.md:88]: take care of indentation

File: workshops/javascript_typing_test/README.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+- A try again button! We can add a try again button to our workshop. Here's the [final demo](https://javascript-typing-test-final-1.gurshanbrar.repl.co/) and [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v1#index.html).
+- Save our results in ```localStorage```. ```localStorage``` is a place on a user's browser to store data. We can store the results there, and fetch them every time we complete the typing test! [final demo](https://javascript-typing-test-v2.gurshanbrar.repl.co/) and [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v2#script.js).
+- Add a nice little navbar. We can add a navbar that links to your github and has a link to Hackclub. Just make sure to read the comment inside the HTML file. [final demo](https://javascript-typing-test-v3.gurshanbrar.repl.co/) and [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v3#index.html).
+
+A version with all of these features is [here](javascript-typing-test.netlify.app)
+
+If you have any questions, you can ping me on slack(@Gurshan) or [my github](https://github.com/gurshanbrar).
```
Comment (giridhar7632) [inline workshops/javascript_typing_test/README.md:324]: You can add the link to your hackclub slack profile

File: workshops/javascript_typing_test/README.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
+---
+name: 'Javascript Typing Test'
+description: 'A Typing Test website built with HTML, CSS, and Javascript'
+author: '@GurshanBrar'
+img: 'https://cloud-gny03pu07.vercel.app/0javascript_typing_test_cover.png'
+---
+
+### First up, the project
```
Comment (giridhar7632) [inline workshops/javascript_typing_test/README.md:8]: Double check the grammar and phrasing

File: workshops/javascript_typing_test/README.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+### Prerequisites
+This project will be more fun to code if you have a general understanding of HTML, CSS, and Javascript. However don't worry if you don't because you can definitely complete this workshop without any prerequisites.
+
+### Focus
+Yes, **focus**. JK. This workshop will focus on Javascript, so you'll have a starter template including HTML and CSS that will be explained.
+
+### The IDE
+An IDE (Integrated Development Environment) is just fancy speak for where your code will run. We'll be using [repl.it](https://repl.it) as our IDE because...  
+1. It saves your code in the cloud
```
Comment (giridhar7632) [inline workshops/javascript_typing_test/README.md:23]: check the ordered lists
they will not render properly on the site

**Review by giridhar7632 — CHANGES_REQUESTED** [review]
Hey! great work 😀
check the grammar and phrasing properly again

**Comment by GurshanBrar** [comment]
### Hi! These changes were requested and committed: 

- Fix grammar and phrasing throughout
- Indent block of code
- Render ordered lists properly
- Add link to slack profile
- Use `<details>` tag

#### Also, thank you for the single backtick suggestion. I will keep that in mind for the future!

File: workshops/javascript_typing_test/README.md [hunk 5]
```diff
@@ -0,0 +1,376 @@
...
+
+- A try again button! We can add a try again button to our workshop. Here's the [final demo](https://javascript-typing-test-final-1.gurshanbrar.repl.co/) and the [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v1#index.html).
+- Save our results in ```localStorage```. ```localStorage``` is a place on a user's browser to store data. We can store the results there, and fetch them every time we complete the typing test! The [final demo](https://javascript-typing-test-v2.gurshanbrar.repl.co/) and the [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v2#script.js).
+- Add a nice little navbar. We can add a navbar that links to your github and has a link to Hackclub. Just make sure to read the comment inside the HTML file. The [final demo](https://javascript-typing-test-v3.gurshanbrar.repl.co/) and the [source code](https://repl.it/@GurshanBrar/Javascript-Typing-Test-v3#index.html).
+
+A version with all of these features is [here](javascript-typing-test.netlify.app)
+
+If you have any questions, you can ping me on [my slack profile](hackclub.slack.com/team/U0198EM6ZA7) or [my github](https://github.com/gurshanbrar).
```
Comment (sampoder) [inline workshops/javascript_typing_test/README.md:376]:
```suggestion
If you have any questions, you can message me on [Slack](hackclub.slack.com/team/U0198EM6ZA7) or [visit my GitHub](https://github.com/gurshanbrar). 
```

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey!! This looks super close, just one thing: could we add more explanation of the JS code starting from `Section 2`? Thanks!

**Pull request closed without merging**